yesterdaysGames.py

```diff
@@ -0,0 +1,60 @@
+"""
+Gray Goolsby
+8/5/2018
+
+Pulls game results from previous day
+"""
+import pandas as pd
+from pullWebData import pullSoup
+
+def pullGameTable(url):
```

Comment:
Maybe consolidate functions- call this `yesterdayGames ():`

no need to give it url if always using same one- just define url on first line of function